pkg/tracing/tracing.go

```diff
@@ -0,0 +1,57 @@
+// Package tracing implements a common tracing initialization for GD2 and CLI
+package tracing
+
+import (
+	"github.com/gluster/glusterd2/glusterd2/gdctx"
+
+	log "github.com/sirupsen/logrus"
+	flag "github.com/spf13/pflag"
+	"go.opencensus.io/exporter/jaeger"
+	"go.opencensus.io/trace"
+)
+
+// Commandline options for exporter endpoints
+const (
+	JaegerEndpointOpt      = "jaegerendpoint"
```

Comment:
Would be better to use '-' as seperator ("jaeger-endpoint" "jaegar-agent-endpoint"). It would make the options much more readable.

Also, with the suggested changes, you don't need to export these constants out of this package.

Reply:
Sure, I have changed the options as suggested and made the constants private to this package.